### PR TITLE
Update run-a-holesky-node.md

### PR DESCRIPTION
**Pull request by amis13**
Adding `cl-shared.yml` to open REST port for beacon RPC in Holesky node
and `L1_BEACON_HTTP` in Taiko node

### CODE REVIEW

**Comment by RogerLamTd** [comment]
@amis13 please resolve conflicts, otherwise lgtm!

File: src/content/docs/guides/run-a-taiko-node.mdx [hunk 1]
```diff
@@ -73,6 +73,7 @@ After getting the address of the Holesky node, set the following L1 node endpoin
 
 - `L1_ENDPOINT_HTTP=http://192.168.1.15:8545`
 - `L1_ENDPOINT_WS=ws://192.168.1.15:8546`
+- `L1_BEACON_HTTP=ws://192.168.1.15:5052`
```
Comment (RogerLamTd) [inline src/content/docs/guides/run-a-taiko-node.mdx:76]: shouldn't this be http?

Comment (amis13) [reply src/content/docs/guides/run-a-taiko-node.mdx]: my bad, yes, I copied the ws and I forgot to change to `http`

**Review by amis13 — COMMENTED** [review]
Thank you @RogerLamTd !
I guess now all it's fine to merge.

**Review by RogerLamTd — APPROVED** [review]

**Review by RogerLamTd — APPROVED** [review]

File: src/content/docs/guides/run-a-holesky-node.md
Comment (amis13) [inline src/content/docs/guides/run-a-holesky-node.md:1]: @RogerLamTd I have changed the numbers, I forgot change them, sorry.

**Comment by amis13** [comment]
@d1onys1us me waiting:

**Review by d1onys1us — APPROVED** [review]

**Comment by d1onys1us** [comment]
> @d1onys1us me waiting:
> 

You don't need to do all that bro, igy

**Comment by amis13** [comment]
> > @d1onys1us me waiting:
> > 
> 
> You don't need to do all that bro, igy

Bro it's just a joke in a comment 😬 damn

**Comment by d1onys1us** [comment]
> > > @d1onys1us me waiting:
> > > 
> > >       
> > >         [
> > >           
> > >         
> > >             
> > >           ](https://private-user-images.githubusercontent.com/119992979/327814144-ef8fea9c-e2b0-4675-b183-f1c911f4f410.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3NzAxNDQsIm5iZiI6MTcxNDc2OTg0NCwicGF0aCI6Ii8xMTk5OTI5NzkvMzI3ODE0MTQ0LWVmOGZlYTljLWUyYjAtNDY3NS1iMTgzLWYxYzkxMWY0ZjQxMC5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwM1QyMDU3MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iY2ExNWIxMzJhZjQyYTg0OGIwNmMxMjNkN2U2M2FiMzczZTViNzgwMTU1NGJlMjRkMmJlOWU2Y2I5MTI2OGQ3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.G7KpACRbk72-sFiTny3sXyyFE5coP2mhurh1b3TxCoo)
> > >         
> > >         
> > >           
> > >             
> > >               
> > >             
> > >             
> > >               
> > >               
> > >             
> > >           
> > >           [
> > >             
> > >               
> > >             
> > >           ](https://private-user-images.githubusercontent.com/119992979/327814144-ef8fea9c-e2b0-4675-b183-f1c911f4f410.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3NzAxNDQsIm5iZiI6MTcxNDc2OTg0NCwicGF0aCI6Ii8xMTk5OTI5NzkvMzI3ODE0MTQ0LWVmOGZlYTljLWUyYjAtNDY3NS1iMTgzLWYxYzkxMWY0ZjQxMC5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwM1QyMDU3MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iY2ExNWIxMzJhZjQyYTg0OGIwNmMxMjNkN2U2M2FiMzczZTViNzgwMTU1NGJlMjRkMmJlOWU2Y2I5MTI2OGQ3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.G7KpACRbk72-sFiTny3sXyyFE5coP2mhurh1b3TxCoo)
> > 
> > 
> > You don't need to do all that bro, igy
> 
> Bro it's just a joke in a comment 😬 damn

it's also a joke lmao

**Comment by amis13** [comment]
> > > > @d1onys1us me waiting:
> > > > 
> > > >       
> > > >         [
> > > >           
> > > >         
> > > >             
> > > >           ](https://private-user-images.githubusercontent.com/119992979/327814144-ef8fea9c-e2b0-4675-b183-f1c911f4f410.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3NzAxNDQsIm5iZiI6MTcxNDc2OTg0NCwicGF0aCI6Ii8xMTk5OTI5NzkvMzI3ODE0MTQ0LWVmOGZlYTljLWUyYjAtNDY3NS1iMTgzLWYxYzkxMWY0ZjQxMC5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwM1QyMDU3MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iY2ExNWIxMzJhZjQyYTg0OGIwNmMxMjNkN2U2M2FiMzczZTViNzgwMTU1NGJlMjRkMmJlOWU2Y2I5MTI2OGQ3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.G7KpACRbk72-sFiTny3sXyyFE5coP2mhurh1b3TxCoo)
> > > >         
> > > >         
> > > >           
> > > >             
> > > >               
> > > >             
> > > >             
> > > >               
> > > >               
> > > >             
> > > >           
> > > >           [
> > > >             
> > > >               
> > > >             
> > > >           ](https://private-user-images.githubusercontent.com/119992979/327814144-ef8fea9c-e2b0-4675-b183-f1c911f4f410.gif?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTQ3NzAxNDQsIm5iZiI6MTcxNDc2OTg0NCwicGF0aCI6Ii8xMTk5OTI5NzkvMzI3ODE0MTQ0LWVmOGZlYTljLWUyYjAtNDY3NS1iMTgzLWYxYzkxMWY0ZjQxMC5naWY_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjQwNTAzJTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI0MDUwM1QyMDU3MjRaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1iY2ExNWIxMzJhZjQyYTg0OGIwNmMxMjNkN2U2M2FiMzczZTViNzgwMTU1NGJlMjRkMmJlOWU2Y2I5MTI2OGQ3JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCZhY3Rvcl9pZD0wJmtleV9pZD0wJnJlcG9faWQ9MCJ9.G7KpACRbk72-sFiTny3sXyyFE5coP2mhurh1b3TxCoo)
> > > 
> > > 
> > > You don't need to do all that bro, igy
> > 
> > Bro it's just a joke in a comment 😬 damn
> 
> it's also a joke lmao

LMAO, You have earned my respect 🥸

**Pull request merged**